app/scripts/controllers/statefulSets.js

```diff
@@ -24,6 +24,7 @@ angular.module('openshiftConsole')
           group: 'apps',
           version: 'v1beta1'
         }, context, function(statefulSets) {
+          $scope.emptyMessage = "No stateful sets to show";
```

Comment:
I'd rather inline the message than use a scope var since it doesn't change. (We do this in other places, but I'd like to get away from that.)